database/seed.js

```diff
@@ -0,0 +1,39 @@
+const Promise = require('bluebird');
+const faker = require('faker');
+const schema = Promise.promisifyAll(require('./data/schemaHelpers'));
+
+// note default properties: id, timestamp, and isliked
+// const sampleSong = {
+//   length: Math.floor(Math.random() * (210 - 100) + 100),
+//   timestamp: 0,
+//   isliked: 0,
+//   songfile: faker.internet.url(),
+//   title: faker.random.words(),
+//   artist: faker.name.findName(),
+//   album: faker.commerce.productName(),
+//   thumbnail: faker.image.imageUrl(),
+// };
+
+// console.log(sampleSong);
+
+const seedData = () => {
+  for (let i = 0; i < 100; i += 1) {
+    let sampleSong = {
+      length: Math.floor(Math.random() * (210 - 100) + 100),
+      timestamp: 0,
+      isliked: Math.floor(Math.random() * 2),
```

Comment:
After looking through documentation for mySQL this makes sense. I did not realize that mySQL does not have full support for actual boolean values. This appears to be a good way of randomizing this then!